packages/root-password-protect/src/plugin/plugin.tsx

```diff
@@ -107,7 +110,9 @@ async function handleProtectedRoute(
       !req.body.password
     ) {
       res.status(400);
-      renderPasswordPage(req, res, {error: 'Bad request (no password).'});
+      renderPasswordPage(req, res, {
+        error: 'Bad request (no password).',
```

Comment:
since we're exposing the password page, we may want to expose these errors as an exported enum/const so that developers can show an appropriate error message for the error.